### ISSUE TITLE
Usage question regarding decryption

### ISSUE THREAD

**Issue by aergonaut** [Shopify/ejson-rails#6]
I am interested in using ejson-rails with one of my projects, but I had one question about usage of the gem.

The README states that the decrypted secrets will be merged into the Rails secrets hash. However, after reading the code of the gem, it doesn't look like ejson-rails will do any actual decrypting of the ejson file itself. My question is, do I have to decrypt the ejson file using the ejson CLI before starting Rails?

**Comment by gmcgibbon** [comment]
👋 Sorry for the delay. Yes, you have to decrypt secrets yourself. This gem only really [reads json files and loads them into your Rails secrets](https://github.com/Shopify/ejson-rails/blob/aec4072b10ced3e6ac2e87f3661ff6569039134b/lib/ejson/rails/railtie.rb#L10-L14). See ejson's [workflow section](https://github.com/Shopify/ejson#workflow), and also [capistrano-ejson](https://github.com/Shopify/capistrano-ejson) for a cap task that does this for you.
